spec/controllers/spree/admin/questions_controller_spec.rb

```diff
@@ -4,7 +4,7 @@
   stub_authorization!
 
   describe '#index' do
-    let!(:question) { create(:question) }
+    let!(:question) { create(:product_question) }
```

Comment:
I think we should also rename the file name of this controller spec..to reflect to change in the name of controller